front/lib/api/assistant/generation.ts

```diff
@@ -91,7 +93,9 @@ export async function renderConversationForModel({
           content: m.content,
         });
       }
-      if (m.action) {
+      // There are contexts where we want to exclude the actions from the rendering.
+      // Eg: During the conversation title generation step.
+      if (m.action && !excludeActions) {
```

Comment:
If this is an action and excludeActions is true we will fall in the else if all the way to the assert never no?

Reply:
I think the excludeAction condition must be inside the that if block?

Reply:
Nevermind me I was one if block off